TemplateBicep/main.bicep

```diff
@@ -0,0 +1,66 @@
+param location string = resourceGroup().location
+param uniqueSolutionPrefix string
+param discoveryZipUrl string
+
+module iotHub 'modules/iothub.bicep' = {
+  name: 'iotHub'
+  params: {
+    name: '${uniqueSolutionPrefix}hub'
+    location: location
+  }
+}
+
+module storage 'modules/storage.bicep' = {
+  name: 'storage'
+  params: {
+    storageAccountName: '${uniqueSolutionPrefix}storage'
+    storageAccountType: 'StorageV2'
```

Comment:
I think these could be switched to internal variables, we don't expect users to switch them

Reply:
Changes added. Please review.

Reply:
LOGTM